syntax = "proto3";

message TestMsgPackage {
  int32 MsgId = 1;
  string MsgName = 2;
}
